libs/printing/src/render.tsx

```diff
@@ -115,8 +115,7 @@ export async function renderToPdf(
         (width - horizontalMargin) * PLAYWRIGHT_PIXELS_PER_INCH
       ),
       height: Math.floor(
-        (PAPER_DIMENSIONS.Letter.height - verticalMargin) *
-          PLAYWRIGHT_PIXELS_PER_INCH
+        (height - verticalMargin) * PLAYWRIGHT_PIXELS_PER_INCH
```

Comment:
Just was looking at the context for this change and noticed that L113 has a misspelling of "noninteger" as "nonintenger." Not related to this change, but maybe fix while we're here looking at it